config/config/wc-config.yaml

```diff
@@ -194,3 +194,8 @@ fluentd:
     tolerations: []
     affinity: {}
     nodeSelector: {}
+velero:
+  veleroNamespaces:
```

Comment:
I'm not a fan of the name `veleroNamespaces`
Maybe something like `includedNamespaces` is more fitting.